src/transport-manager.js

```diff
@@ -215,6 +226,26 @@ class TransportManager {
 
     await Promise.all(tasks)
   }
+
+  /**
+   * Create self signed peer record raw envelope.
+   * @return {Uint8Array}
+   */
+  async _createSelfPeerRecord () {
+    try {
+      const peerRecord = new PeerRecord({
+        peerId: this.libp2p.peerId,
+        multiaddrs: this.libp2p.multiaddrs
+      })
+      const envelope = await Envelope.seal(peerRecord, this.libp2p.peerId)
+      this.libp2p.peerStore.addressBook.consumePeerRecord(envelope)
+
+      return this.libp2p.peerStore.addressBook.getRawEnvelope(this.libp2p.peerId)
```

Comment:
There's no need to return anything, nothing is consuming the result of this, so we should always return null.

nit: Calling this function `_updateSelfPeerRecord` might be a bit clearer about what it's doing.